docs/Advanced.md

```diff
@@ -1,5 +1,9 @@
 # Advanced topics
 
+## Updating the server
```

Comment:
I feel like this belongs to `Basics` 🤔

Reply:
Maybe it belongs to the CLI commands and just that? i did think about that too but it seems out of place in Basics.. ?